components/portal/widgets/controllers.js

```diff
@@ -380,21 +380,32 @@ define(['angular', 'moment'], function(angular, moment) {
         .then(function(data) {
           // Make sure we got a something back
           if (data) {
-            // Add an action item to scope array
-            $scope.actionItems.push({
-              textSingular: item.textSingular,
-              textPlural: item.textPlural,
-              actionUrl: item.actionUrl,
-              quantity: data.quantity,
-            });
+            // Make sure quantity is a number
+            if (isNaN(data.quantity)) {
+              // Log problem and don't include in list
+              $log.warn('Got non-number data from ' + item.feedUrl);
+              // Reduce display limit (only once) to make room for error
+              if (!$scope.errorQuantity) {
+                $scope.itemsLimit -= 1;
+              }
+              // Show error
+              $scope.errorQuantity = true;
```

Comment:
I'd have expected `errorQuantity` to be a quantity rather than a boolean.

Reply:
I'm fine with changing this.